raiden_contracts/deploy/__main__.py

```diff
@@ -73,10 +74,12 @@ def __init__(
         self.precompiled_path = contracts_precompiled_path(self.contracts_version)
         self.contract_manager = ContractManager(self.precompiled_path)
 
-        # Check that the precompiled data is correct
-        contract_manager_source = ContractManager(contracts_source_path())
-        contract_manager_source.checksum_contracts()
-        contract_manager_source.verify_precompiled_checksums(self.precompiled_path)
+        # Check that the precompiled data matches the source code
+        # Only for current version, because this is the only one with source code
```

Comment:
Why do we have old precompiled data (without sources) around?

Reply:
I guess this question is is out of scope of this PR.

Reply:
Because the old precompiled data was added on the fly and it was considered to be a temporary addition just for the Raiden client tests with no limits.

We need to solve this for following versions.